# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../tvtime/compile-fix.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur tvtime-1.0.2-orig/src/videodev2.h tvtime-1.0.2/src/videodev2.h
--- tvtime-1.0.2-orig/src/videodev2.h	2008-02-26 20:28:56.000000000 +0100
+++ tvtime-1.0.2/src/videodev2.h	2008-02-26 20:29:40.000000000 +0100
@@ -16,7 +16,6 @@
 #ifdef __KERNEL__
 #include <linux/time.h> /* need struct timeval */
 #endif
-#include <linux/compiler.h> /* need __user */
 
 /* for kernel versions 2.4.26 and below: */
 #ifndef __user
